src/autorun.ts

```diff
@@ -69,7 +94,12 @@ export function run<T>(fn: Cb<T>): Observable<T> {
     function createTracker(track: boolean) {
         return function $<O>(o: Observable<O>): O {
             if (deps.has(o)) {
-                const v = deps.get(o);
+                const v = deps.get(o)!;
+                if (track && !v.track) {
+                    // Previously tracked with _, but now also with $.
+                    // So completed state becomes relevant now.
+                    v.track = true;
```

Comment:
Didn't get that at first. Does this cover a case of `run(() => _(o) + $(o))` ? I haven't considered that! 👍
Can you add this code to the comment as well, so that the case would be instantly obvious?